File: deamhome/src/main/java/sosteam/deamhome/global/attribute/Token.java
package sosteam.deamhome.global.attribute;

import lombok.Getter;

@Getter
public enum Token {
    ACCESS(
            1000 * 60 * 60 * 12,
            "accessToken"
    ),
    REFRESH(
            1000 * 60 * 60 * 24 * 14,
            "refreshToken"
    );

    private final Long milliseconds;
    private final String type;

    Token(int time, String type) {
        this.milliseconds = Long.valueOf(time);
        this.type = type;
    }
}

File: src/main/kotlin/sosteam/deamhome/global/attribute/Token.kt
package sosteam.deamhome.global.attribute

enum class Token(val time: Long, val type: String) {
	ACCESS(
		60 * 60 * 4,
		"accessToken"
	),
	REFRESH(
		60 * 60 * 24 * 14,
		"refreshToken"
	)
}